notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        THETA = np.radians(-28.904)
</pre><p>This seems a little arbitrary to me as it's not strictly true for all streets, I would omit it but it's up to you.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>I think it's ok to leave in, it shouldn't make much of a difference and is an interesting example.</p>